Exercise_1.py

```diff
@@ -1,19 +1,22 @@
 class myStack:
      def __init__(self):
-         
+          self.arr = []
      def isEmpty(self):
-         
+          if len(self.arr) == 0:
+               return True
+          else:
+               return False
      def push(self, item):
-         
+          self.arr.append(item)
      def pop(self):
```

Comment:
Take care of the edge cases while popping and peeking the elements when the stack is empty. 

In this case, I get an index out of bound error

Reply:
resolved and pushed